moler/cmd/at/set_mode.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+"""
+AT+COPS
+
+AT commands specification:
+google for: 3gpp specification 27.007
+(always check against latest version of standard)
+"""
+
+__author__ = 'Adam Klekowski'
+__copyright__ = 'Copyright (C) 2020, Nokia'
+__email__ = 'adam.klekowski@nokia.com'
+
+import re
+
+from moler.cmd.at.genericat import GenericAtCommand
+from moler.exceptions import ParsingDone
+
+
+class SetMode(GenericAtCommand):
+    """
+    Command to set mode (automatic, lte, gsm, wcdma).
+    """
+    mode2cops_value = {"automatic": '0',
+                       "lte": '0,0,0,7',
+                       "gsm": '0,0,0,3',
+                       "wcdma": '0,0,0,6'}
+
+    def __init__(self, selected_mode, connection=None, prompt=None, newline_chars=None, runner=None):
+        """Create instance of SetMode class"""
+        super(SetMode, self).__init__(connection, operation='execute', prompt=prompt,
+                                      newline_chars=newline_chars, runner=runner)
+        self.selected_mode = selected_mode.lower()
+
+        if self.selected_mode not in SetMode.mode2cops_value:
+            raise ValueError('\"{}\" is not correct mode. Available modes: {}.'.format(
```

Comment:
1. Please use MolerException and its subclasses. I think 'WrongUsage' fits best.
2. Not covered by tests.